### PR TITLE
Update Why CachyOS? section

### PR DESCRIPTION
**Pull request by Aarrayy**
I think for now its way better with the new wording.

I also removed some things such as RT being labeled as a scheduler

Feel free to review and make a suggestion if needed

### CODE REVIEW

File: src/content/docs/cachyos_basic/why_cachyos.md [hunk 1]
```diff
@@ -3,11 +3,13 @@ title: Why CachyOS?
 description: Why CachyOS may be better for you
 ---
 
-CachyOS is an user-friendly and highly optimized Linux distribution based on Arch Linux.
+CachyOS is a user-friendly and highly optimized Linux distribution based on Arch Linux.
 
-# What makes CachyOS not just Arch based distro.
+## What makes CachyOS not just another Arch based distro?
```
Comment (1Naim) [inline src/content/docs/cachyos_basic/why_cachyos.md]: This line seems a bit out of place, maybe we can merge it with the bottom paragraph

Comment (1Naim) [inline src/content/docs/cachyos_basic/why_cachyos.md]: After moving the above sentence to the bottom paragraph, maybe we can just remove this subheading altogether. If removing this makes the below paragraph lose context, we can change the title of this page. What do you think?

**Review by 1Naim — CHANGES_REQUESTED** [review]

File: src/content/docs/cachyos_basic/why_cachyos.md [hunk 2]
```diff
@@ -3,41 +3,41 @@ title: Why CachyOS?
...
 
 ## Customizable Installation Process
-When you boot into the live ISO from the [bootable USB](/installation/installation_prepare/#creating-a-bootable-cachyos-usb-drive) that you created,
-you will automatically meet our installer. Our installer provides much more choice than other distributions.
-We offer a variety of [desktop environments](/installation/desktop_environments/), [boot loaders](/installation/bootmanagers/)
-and [kernels](kernel/kernel/#variants) to choose from.
-You can and should choose what you want to have installed and what not to have installed. Your system is your home.
+Start your CachyOS journey by booting from the [bootable USB](/installation/installation_prepare/#creating-a-bootable-cachyos-usb-drive) that you created,
+You’ll be welcomed by our Hello program, a helpful introduction to CachyOS. Our customized Calamares installer provides a wide range of possibilities.
+We offer a variety of [desktop environments](/installation/desktop_environments/), [boot managers](/installation/boot_managers/)
```
Comment (Aarrayy) [inline src/content/docs/cachyos_basic/why_cachyos.md:30]: boot managers link should work after #53 is merged

File: src/content/docs/cachyos_basic/why_cachyos.md [hunk 3]
```diff
@@ -3,41 +3,41 @@ title: Why CachyOS?
...
+There are x86-64-v3, x86-64-v4 and Zen4 optimized repositories that exists to enhance your experience by
 reducing latency, improving performance, applying special fixes, etc.
 
-:::caution[WARNING]
-x86-64-v3 repository only works for CPUs that supports AVX and AVX2 instructions
-while x86-64-v4 only works for AVX and AVX2 + AVX512 instructions that is supported on AMD CPUs
-from Zen4 and higher. CachyOS automatically selects which repositories are supported by checking your CPU, so you should not
-change it.
-:::
+For more information, check out [this](/cachyos_repositories/what_are_the_cachyos_repo) page for a more detailed explanation of the optimized repositories that we provide.
```
Comment (1Naim) [inline src/content/docs/cachyos_basic/why_cachyos.md]:
```suggestion
Check out [this](/cachyos_repositories/what_are_the_cachyos_repo) page for a more detailed explanation of the optimized repositories that we provide.
```

**Review by 1Naim — CHANGES_REQUESTED** [review]

File: src/content/docs/cachyos_basic/why_cachyos.md (hunk 3 above)
Comment (Aarrayy) [reply src/content/docs/cachyos_basic/why_cachyos.md]: u sure?

Comment (Aarrayy) [reply src/content/docs/cachyos_basic/why_cachyos.md]: nvm i see why xD

**Review by 1Naim — APPROVED** [review]
LGTM

**Review by ptr1337 — APPROVED** [review]

**Pull request merged**